tests/ci/install_environment.sh

```diff
@@ -1,29 +1,31 @@
 #!/usr/bin/env bash
 
-export NUOVERSION=4.0.7.1
+export NUOVERSION=5.0.3.3
+export TAR_VERSION=nuodb-${NUOVERSION}.linux.x86_64
 export INFLUX_VERSION=1.4.3
-
-sudo sh -c 'echo madvise > /sys/kernel/mm/transparent_hugepage/enabled'
-sudo sh -c 'echo madvise > /sys/kernel/mm/transparent_hugepage/defrag'
-wget -q http://ce-downloads.nuohub.org/nuodb-ce_${NUOVERSION}_amd64.deb --output-document=/var/tmp/nuodb.deb
-sudo dpkg -i /var/tmp/nuodb.deb
+echo "http://ce-downlaods.nuohub.org/${TAR_VERSION}.tar.gz"
```

Comment:
You can drop this print statement. If you want to see what the script is doing, you can invoke `set -x` at the beginning of the script.